### PR TITLE
feat(action): adds flag to optionally fail on errors

### PR DESCRIPTION
**Pull request by cmitzel-ncino**
WIP, trying to implement a flag to the action such that the action will result in a success status regardless of errors or warnings. 

I need to implement one or two tests still, I'm pretty new to developing in TS outside of the AWS CDK so taking my time to learn.

### CODE REVIEW

**Comment by cmitzel-ncino** [comment]
linked with issue 687 https://github.com/wagoid/commitlint-github-action/issues/687

**Comment by cmitzel-ncino** [comment]
@wagoid I added a couple tests and some docs, let me know if this needs anything else!

**Review by wagoid — COMMENTED** [review]
Awesome work @cmitzel-ncino! Could you also add this param to `action.yml` file? Like [here](https://github.com/wagoid/commitlint-github-action/blob/master/action.yml#L17-L20)

File: src/action.test.js [hunk 1]
```diff
@@ -110,10 +110,49 @@ describe('Commit Linter action', () => {
 
     await runAction()
 
+    console.log()
```
[the remark this answers is not in the document]
Comment (cmitzel-ncino) [reply src/action.test.js:113]: https://tenor.com/view/the-s-impsons-bush-homer-simpsons-homer-hiding-gif-4325800

**Comment by cmitzel-ncino** [comment]
Hitting an issue with the action reporting 
```Passing despite errors ✅
Error: Unable to process file command 'output' successfully.
Error: Invalid format 'You have commit messages with errors
```

Looking at the [docs](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#writing-the-action-code) I believe I'm missing an argument for the `setOutput()` function so I'm rectifying that now.

**Pull request merged**